src/nodejs/supply/supply_test.go

```diff
@@ -279,8 +279,6 @@ var _ = Describe("Supply", func() {
 				defer os.Remove(nvmrcFile)
 
 				testCases := [][]string{
-					{"lts/fermium", "14.*.*"},
-					{"lts/gallium", "16.*.*"},
```

Comment:
Should we be keeping `16.*.*` here until its deprecation date?

Reply:
Good catch, I rollback that line